src/Tizen.NUI/src/public/BaseComponents/VideoView.cs

```diff
@@ -383,6 +383,31 @@ public void Backward(int millisecond)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Get native player handle.
+        /// <example>
+        /// How to get native player handle
+        /// <code>
+        /// VideoView videoView = new VideoView();
+        /// videoView.ResourceUrl = "some xxx video path";
```

Comment:
even though I use the "xxx" as random placeholder myself, please remove this from public API, we don't want to have phrase "xxx video" here :)
```suggestion
        /// videoView.ResourceUrl = "some video path";
```